### PR TITLE
gts3a.cpp: Dumped and added a new 'Wipeout' set

### PR DESCRIPTION
**Pull request by clawgrip**
Dumped and added a new 'Wipeout' set. 
Also:
* Made the newest revisions to be the parents sets (some games on this driver had the newest, but others the oldest) and reorder de ROM loads accordingly (newer to older).
* Reformatted single line comments.
* Renamed some sets to follow the driver name scheme and to avoid confusion with other games on different drivers.

New NOT_WORKING clones
-----------------------------------
Wipeout (rev.2, alt) [Victor Fernandez (City Game)]

### CODE REVIEW

**Comment by einstein95** [comment]
To quote [Carny_Priest on VPForums](https://www.vpforums.org/index.php?showtopic=36205) 
> Sys3 - Cue Ball Wizard, Lights, Camera, Action, Street Fighter II, Super Mario Bros.,  - No rev numbers but there are on-display gprom checksum test results that can be used to distinguish one set from another. Just would not be able to tell which one is updated.

These should probably be kept in their previous order but renamed to "set 1", "set 2" etc.

Similar to the newly added alt of Wipeout, "World Challenge Soccer (rev.1, disp.rev.2)" should be renamed to be an alt of Wipeout. The differing rom doesn't seem to have any explicit revision strings, but does seem to use some fields that are set as "NOT USED" in `wipeout`.

**Comment by clawgrip** [comment]
Done!
 Better now, thanks.

File: src/mame/drivers/gts3a.cpp [hunk 1]
```diff
@@ -1318,43 +1340,44 @@ ROM_END
...
+GAME(1995,  stargatp2,  stargatp, p7, gts3a, gts3a_state, init_gts3a, ROT0, "Gottlieb", "Stargate (pinball, rev.2)",                  MACHINE_IS_SKELETON_MECHANICAL | MACHINE_SUPPORTS_SAVE )
+GAME(1995,  stargatp1,  stargatp, p7, gts3a, gts3a_state, init_gts3a, ROT0, "Gottlieb", "Stargate (pinball, rev.1)",                  MACHINE_IS_SKELETON_MECHANICAL | MACHINE_SUPPORTS_SAVE )
+GAME(1995,  stargatp0,  stargatp, p7, gts3a, gts3a_state, init_gts3a, ROT0, "Gottlieb", "Stargate (pinball)",                         MACHINE_IS_SKELETON_MECHANICAL | MACHINE_SUPPORTS_SAVE )
+GAME(1995,  shaqattq,   0,        p7, gts3a, gts3a_state, init_gts3a, ROT0, "Gottlieb", "Shaq Attaq (rev.5)",                         MACHINE_IS_SKELETON_MECHANICAL | MACHINE_SUPPORTS_SAVE )
+GAME(1995,  shaqattq2,  shaqattq, p7, gts3a, gts3a_state, init_gts3a, ROT0, "Gottlieb", "Shaq Attaq (rev.2)",                         MACHINE_IS_SKELETON_MECHANICAL | MACHINE_SUPPORTS_SAVE )
+GAME(2003,  mac_zois,   shaqattq, p7, gts3a, gts3a_state, init_gts3a, ROT0, "Aksioma",  "Machina Zois Virtual Training Centre",       MACHINE_IS_SKELETON_MECHANICAL | MACHINE_SUPPORTS_SAVE )
```
Comment (einstein95) [inline src/mame/drivers/gts3a.cpp:1371]: While you're at it, this should be titled "machinaZOIS Virtual Training Center" according to the [developer's page on the game](https://aksioma.org/machinazois_vtc/)

Comment (clawgrip) [reply src/mame/drivers/gts3a.cpp]: Fixed, thanks!

**Pull request merged**